### PR TITLE
fix(edgeless): optimize ai panel user experience

### PR DESCRIPTION
**Pull request by fundon**
Closes: AFF-902,AFF-901,AFF-908

### CODE REVIEW

**Comment by fundon** [comment]
* **#6810** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/6810?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @fundon and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/blocks/src/root-block/edgeless/components/toolbar/edgeless-toolbar.ts [hunk 1]
```diff
@@ -348,6 +349,16 @@ export class EdgelessToolbar extends WithDisposable(LitElement) {
     }
   }
 
+  // The toolbar should be disabled while edgeless AI is in progress.
```
Comment (doouding) [inline packages/blocks/src/root-block/edgeless/components/toolbar/edgeless-toolbar.ts:352]: I think this part should be moved to the edgeless copilot widget.  Get the reference through `host.view.getWidget` and change its style or something.

**Review by doouding — APPROVED** [review]

File: packages/blocks/src/root-block/edgeless/components/toolbar/shape/shape-tool-button.ts [hunk 2]
```diff
@@ -75,10 +75,7 @@ export class EdgelessShapeToolButton extends EdgelessToolButton<
     }
 
     .container-clip {
-      padding: 10000px 10000px 0;
```
Comment (CatsJuice) [inline packages/blocks/src/root-block/edgeless/components/toolbar/shape/shape-tool-button.ts:78]: Leds to some unexpected effects #7139

Comment (fundon) [reply packages/blocks/src/root-block/edgeless/components/toolbar/shape/shape-tool-button.ts:78]: A very large layer will be created here.

Comment (CatsJuice) [reply packages/blocks/src/root-block/edgeless/components/toolbar/shape/shape-tool-button.ts:78]: Yes, that's expected, to achieve the effect of `overflow: clip` only on the bottom edge. Is there any performance issue or other problem with this approach?

Comment (fundon) [reply packages/blocks/src/root-block/edgeless/components/toolbar/shape/shape-tool-button.ts:78]: The layer is always there. 

You can consider trying canvas overlayer, also create a separate element outside(fixed/absolute).

Comment (fundon) [reply packages/blocks/src/root-block/edgeless/components/toolbar/shape/shape-tool-button.ts:78]: Screenshot 2024-05-26 at 12 45 37

Comment (fundon) [reply packages/blocks/src/root-block/edgeless/components/toolbar/shape/shape-tool-button.ts:78]: > Yes, that's expected, to achieve the effect of `overflow: clip` only on the bottom edge

clip does not work on padding

**Pull request merged**